region/maridia/inner-yellow/Maridia Elevator Room.json

```diff
@@ -277,6 +277,121 @@
       "flashSuitChecked": true,
       "devNote": "This is mainly useful as an alternative to crouch-jump down-grab in order to preserve a flash suit."
     },
+    {
+      "link": [1, 2],
+      "name": "Side Platform Cross Room Jump",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 7.4375,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [],
+              "note": ["This applies to Warehouse Entrance."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 8.4375,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [],
+              "note": ["This applies to Ridley Tank Room, Halfie Climb Room, and Dust Torizo Room."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 8.4375,
+              "speedBooster": true,
+              "obstructions": [[2, 0]],
+              "requires": [
+                "canMomentumConservingMorph"
+              ],
+              "note": ["This applies to Early Supers Room and Waterway Energy Tank Room."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 20.4375,
+              "speedBooster": true,
+              "obstructions": [[2, 0]],
+              "requires": [
+                "canMomentumConservingTurnaround"
+              ],
+              "note": ["This applies to Waterway Energy Tank Room."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 19.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "requires": [
+                "canMomentumConservingMorph"
+              ],
+              "note": ["This applies to Noob Bridge, Metroid Room 1, and Statues Hallway."]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 39.4375,
```

Comment:
I count 38 tiles 🤔 
And it should be displayed more clearly to not use the full runway.

Reply:
Yeah, I had the length wrong, went through and fixed it everywhere now.

Also updated the "minTiles" in this strat to reflect the recommended length, and added more detail to the note.